web_cache_name_get/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Raphaël Reverdy - Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Cache name_get",
+    "version": "10.0.1.0.0",
+    "author": "Akretion, "
```

Comment:
@hparfr Added website key as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)